### PR TITLE
(DOCSP-17079): Mac Catalyst Support - React Native

### PR DESCRIPTION
**Pull request by mongomoe**
### Jira

- https://jira.mongodb.org/browse/DOCSP-17079

### Staged Changes (Requires MongoDB Corp SSO)

- [React Native SDK > Install](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-17079-macOS-ReactNative-support/sdk/react-native/install/)
- [React Native SDK > Integrations > Build for Mac Catalyst](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-17079-macOS-ReactNative-support/sdk/react-native/integrations/mac-catalyst/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/includes/steps-mac-catalyst.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+title: Update Your Build Phase Settings
```
Comment (takameyer) [inline source/includes/steps-mac-catalyst.yaml:1]: Just want to make clear, this step should go away in the next React Native release.  Here is the relevant PR https://github.com/facebook/react-native/pull/31062

Comment (takameyer) [reply source/includes/steps-mac-catalyst.yaml:1]: So maybe we could already cover our bases by saying do this step for RN < v0.65

**Review by MongoCaleb — APPROVED** [review]
LGTM

File: source/includes/steps-mac-catalyst.yaml (hunk 1 above)
Comment (mongomoe) [reply source/includes/steps-mac-catalyst.yaml:1]: Thanks 👍  added some text in a few spots saying that this is for RN v 0.64 and below, and that you don't need it for RN v 0.65 and above.

**Pull request merged**